style.css

```diff
@@ -5,18 +5,37 @@
     'FILL' 0,
     'wght' 700,
     'GRAD' 0,
-    'opsz' 48;
+    'opsz' 58;
 }
 
 * {
   margin: 0;
+
+  /* font-family: 'Playfair Display', serif; */
+  font-family: 'Poppins', sans-serif;
 }
 
 h1 {
   font-size: 4.8em;
   font-family: 'Playfair Display', serif;
   line-height: 1em;
   margin: 0 0 0.4em 0;
+  font-weight: 200;
+}
+
+h2 {
+  font-size: 4.8em;
+  font-family: 'Playfair Display', serif;
```

Comment:
- [ ] It is noticed that ` font-family: 'Playfair Display', serif;` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.pry-font`  and assigning it wherever possible as a class name in your `html` markup code.
```
.pry-font {
  font-family: 'Playfair Display', serif;
}
```
- And kindly attend to any repetitions in your code 🙂.